### PR TITLE
Add enrich data app example

### PR DESCRIPTION
**Pull request by ericcheatham**

### CODE REVIEW

File: enrich/fixtures/pg.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "users": [
+    {
+      "key": "100",
+      "value": {
+        "id": "100",
+        "username": "ahamidi",
+        "email": "ahamidi@me.com"
```
Comment (ahmeroxa) [inline enrich/fixtures/pg.json:8]:
```suggestion
        "email": "ali@meroxa.io"
```

**Comment by ahmeroxa** [comment]
@ericcheatham can we get this ready to be merged soon?

File: ecommerce-order-alerting/main.py [hunk 2]
```diff
@@ -22,13 +21,9 @@
...
 
 def send_alert(records: t.List[Record]) -> t.List[Record]:
-    updated = []
     for record in records:
         try:
-            if isinstance(record.value, str):
-                payload = json.loads(record.value)["payload"]
-            else:
-                payload = record.value["payload"]
+            payload = record.value["payload"]
```
Comment (ericcheatham) [inline ecommerce-order-alerting/main.py:26]: [turbine-py 1.0.5](https://github.com/meroxa/turbine-py/releases/tag/v1.0.5) addressed an issue wherein records were not being properly serialized/deserialized

File: python-sample-data-app/requirements.txt [hunk 3]
```diff
@@ -1,3 +1 @@
-# will include meroxa-py once published
-autopep8==1.6.0
```
Comment (ericcheatham) [inline python-sample-data-app/requirements.txt:2]: `autopep8` is not needed here. `aiottp` is pulled in as a dependency of `turbine-py`

File: enrich/main.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    return records
+
+
+class App:
+    @staticmethod
+    async def run(turbine: Runtime):
+
+        logging.basicConfig(level=logging.INFO)
+
+        try:
```
Comment (janelletavares) [inline enrich/main.py:31]: register clearbit env var secret in this block

File: enrich/main.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import logging
+import typing as t
+
+from turbine.runtime import Record, Runtime
+
+from enrich import enrich_user_email
+
+
+def enrich_data(records: t.List[Record]) -> t.List[Record]:
+    for record in records:
```
Comment (janelletavares) [inline enrich/main.py:10]: a try, just to be extra safe

**Review by janelletavares — APPROVED** [review]

**Pull request merged**